pkg/controller/core/workload_controller_test.go

```diff
@@ -176,3 +178,127 @@ func TestAdmittedNotReadyWorkload(t *testing.T) {
 		})
 	}
 }
+
+func TestSyncCheckConditions(t *testing.T) {
+	testStartTime := metav1.NewTime(time.Now())
```

Comment:
```suggestion
	now := metav1.NewTime(time.Now())
```

for consistency with other tests

Reply:
done